scripts/overrides/tei_to_html.xsl

```diff
@@ -118,10 +122,40 @@
         <xsl:text>.</xsl:text>
       </li>
 
-      <li><strong>Contributors to digital file: </strong> <xsl:value-of separator=", " select="//teiHeader/fileDesc/notesStmt/note/persName"></xsl:value-of></li>
+    <!-- editorial note(s) field -->
+    <xsl:if test="//body//note[@type='editorial'] | //body//note[@type='authorial'][@resp='unk'] | //notesStmt/note | //sourceDesc/bibl/note[@type='project'][not(@target)]">
+        <xsl:choose>                        
+            <xsl:when test="count(//body//note[@type='editorial'] | //body//note[@type='authorial'][@resp='unk']) = 1">
+                <li><strong>Editorial Note: </strong>
+                    <xsl:if test="//body//note[@type='editorial']">
+                        <xsl:value-of select="//body//note[@type='editorial']"/>
+                    </xsl:if>
+                    <xsl:if test="//body//note[@type='authorial'][@resp='unk']">
+                        <xsl:text>The marginal annotation, "</xsl:text><xsl:apply-templates select="//body//note[@type='authorial'][@resp='unk']"/><xsl:text>," is in a hand other than Whitman's.</xsl:text>
+                    </xsl:if>
+                </li>
+            </xsl:when>
+            
+            <xsl:when test="count(//body//note[@type='editorial']) > 1">
+                <li><strong>Editorial Notes: </strong>
+                    <xsl:for-each select="//body//note[@type='editorial']">
+                        <xsl:number level="any" count="note[@type='editorial']"/>
+                        <xsl:text>. </xsl:text><xsl:apply-templates/>
+                        <xsl:if test="following::note[@type='editorial']"><br /><br /></xsl:if>
+                    </xsl:for-each>
+                </li>
+            </xsl:when>                                
+        </xsl:choose>
+    </xsl:if>
+
+       
+      <li><strong>Contributors to digital file: </strong> <xsl:value-of separator=" | " select="//teiHeader/fileDesc/notesStmt/note/persName"></xsl:value-of></li>
```

Comment:
@karindalziel is it possible to swap this out for your new Contributors method? I realize I might have missed that step, but I also couldn't remember if that code would apply to this `<persName>` way of listing contributors in the TEI

Reply:
(The TEI has this structure)

```
<notesStmt>
        <note type="project">The following are responsible for particular readings or for
          changes to this file, as noted: <persName xml:id="el">Elizabeth Lorang</persName>
          <persName xml:id="js">John Schwaninger</persName>
          <persName xml:id="nnk">Nima Najafi Kianfar</persName>
          <persName xml:id="km">Kevin McMullen</persName>
          
        </note>
      </notesStmt>
```